letmein/src/command/close.rs

```diff
@@ -0,0 +1,225 @@
+// -*- coding: utf-8 -*-
```

Comment:
I am wondering, if we can unify the close and knock implementations on the client side and create one common "Protocol sequence" implementation for both.
They are basically identical. The only difference should only be the `Operation::Close`.